src/calendar/day/index.js

```diff
@@ -82,7 +82,7 @@ export default class Day extends Component {
     const {day} = this.props;
     const date = xdateToData(day);
     const Component = this.getDayComponent();
-    const dayProps = extractComponentProps(Component, this.props);
+    const dayProps = extractComponentProps(Day, this.props);
```

Comment:
You are in the Day component which means I can use this.props and not extractOwnProps function. This is meant to extract the props of the relevant component, whether it is BasicDay, PeriodDay, or your custom dayComponent, from this.props and pass only them (without Day extra props).

Reply:
nice

Reply:
@Inbal-Tish 
Is this the correct answer?
```
const dayProps = this.props
```

Reply:
Yes but you don't need the 'dayProps` anymore, you can just spread `this.props`